app/src/main/java/timur/gilfanov/messenger/ui/screen/settings/ProfileUiState.kt

```diff
@@ -0,0 +1,6 @@
+package timur.gilfanov.messenger.ui.screen.settings
+
+sealed interface ProfileUiState {
```

Comment:
Must be declared as immutable, because it's aligned with MVI architecture, and make sure class stability for Compose.
```suggestion
@Immutable
sealed interface ProfileUiState {
```